lib/core/themes/app_colors.dart

```diff
@@ -7,4 +7,11 @@ class AppColors {
   static const Color deepGrey = Color(0xff3c3c3c);
   static const Color black = Color(0xff000000);
   static const Color white = Color(0xffffffff);
+  static const Color textGradientColor1 = Color(0xFFA64EDD);
```

Comment:
that's name not clear ( textGradientColor1 ,2 ,3 )

u can make it better like ( loginTextGradientBlue , loginTextGradientGreen , loginTextGradientWhite )

that would be better.